Banyan/src/requests.jl

```diff
@@ -208,7 +225,10 @@ function partitioned_computation(fut::AbstractFuture; destination, new_source=no
             message = receive_next_message(gather_queue, p)
             message_type = message["kind"]
             if message_type == "JOB_READY"
-                # @debug "Job $job_id is ready"
+                @info "Job $job_id is running"
```

Comment:
Debug mode only?

Reply:
I think this is fine, because since the JOB_READY message only gets sent once, this will only happen the first time the job is running.
Also, I think this case will never happen, since the JOB_READY message is read from the backend.

Reply:
I still don't think we should have this - we already have a debug message in `wait_for_job`. If anything, we would be changing that from `@debug` to `@info`. But the thing is that we already have a progress bar that shows progress of the bar waiting if it takes more than a couple seconds. So I think we should just stick to the progress bar and not have any extra print statements. What do you think?

Reply:
sounds good, thanks!